### DISCUSSION TITLE
Button style is different from example of doc

### DISCUSSION THREAD

**Discussion by tylerrrkd** [tamagui/tamagui#1190] [General]
### Current Behavior

I use the **starter** to create my project by `npx create tamagui@latest`, and I use the default theme from `@my/ui`.

And it is weird that the `light` theme show even not button frame color, which different from doc's example.

expected:

wanna try an example like doc.

![83A2A773-929F-412E-9E2A-98EF4D3DE3F2](https://github.com/tamagui/tamagui/assets/8706896/3313e91c-fc53-4b46-abd5-e64a2c70e167)

![5C2DBF72-3DFE-4A72-B0D2-B68D0435EEDD_1_105_c](https://github.com/tamagui/tamagui/assets/8706896/d6409ebe-ef5c-4e78-b608-23049c67d200)

![C2C9ADFF-FC68-4D24-81FD-1BFA47D9085C_1_105_c](https://github.com/tamagui/tamagui/assets/8706896/4879793c-e2ed-4f14-9a6a-c1049c3deeaa)

![C9B7D856-2E53-4041-89B9-4C3EF8E45C61_1_105_c](https://github.com/tamagui/tamagui/assets/8706896/2d02053c-b2ec-4218-9680-b5b8b03445c7)

### Tamagui Version

```markdown
@latest
```

### Reproduction

```markdown
// default theme and project directory which create by starter

// code below


import { Button, XStack, YStack } from '@my/ui'
import React from 'react'
import { useSafeAreaInsets } from 'react-native-safe-area-context'

export function HomeScreen() {
  const { bottom } = useSafeAreaInsets()

  return (
    <YStack f={1} p="$4" space>
      <YStack f={1} jc="center" ai="center">
        <Button circular>start</Button>
      </YStack>
      <YStack bottom={bottom}>
        <XStack>
          <Button f={1}>file list</Button>
        </XStack>
      </YStack>
    </YStack>
  )
}
```

**Comment by natew** [comment]
The button background is a light grey is it just that your background is light gray so its not showing? What happens if you put it on a pure white background? you may need to adjust the theme

**Comment by tylerrrkd** [comment]
@natew I have change the `theme` in [this file](https://github.com/tamagui/tamagui/blob/master/starters/next-expo-solito/packages/ui/src/tamagui.config.ts#L63). But it doesn't work. It seems that `Tamagui` really need a doc like [Antd](https://ant.design/docs/react/customize-theme#api) does,  which really clearly shows what COLORs and SPACINGs will effect what component in global. As you can see the issues on github and questions on discord are frequently mentioning about `global theme` problem. plz.
